CONTRIBUTING.md

```diff
@@ -311,9 +311,11 @@ To save @bors some work, and to get small changes through more quickly, when
 the other rollup-eligible patches too, and they'll get tested and merged at
 the same time.
 
-To find documentation-related issues, sort by the [A-docs label][adocs].
+To find documentation-related issues, sort by the [T-doc label][tdoc].
+You can find documentation style guidelines in [RFC 1574][rfc1574].
```

Comment:
Could you put a newline above this? Markdown means that this will be one paragraph, let's make it two 😄